src/Microsoft.TemplateEngine.Cli/TemplateInvoker.cs

```diff
@@ -256,12 +258,10 @@ private async Task<CreationResultStatus> CreateTemplateAsync(ITemplateMatchInfo
                         string fixedMessage = string.Join(", ", missingParamNamesCanonical);
                         Reporter.Error.WriteLine(string.Format(LocalizableStrings.MissingRequiredParameter, fixedMessage, resultTemplateName).Bold().Red());
                     }
-                    break;
-                case CreationResultStatus.OperationNotSpecified:
```

Comment:
Shouldn't we return `OperationNotSpecified` instead of `UnexpectedResult`?

Reply:
I feel like `OperationNotSpecified` should be used when unexpected operation comes from CLI itself and not the unexpected result comes from template engine core.